spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -931,4 +931,13 @@ def fixture_record(fixture_name, indexer: @indexer)
       expect(sample).to be_nil
     end
   end
+
+  context "Temporary in resource sharing location" do
+    it "does not show as a temporary location" do
+      indexer = IndexerService.build
+      sample = indexer.map_record(fixture_record('998370993506421', indexer: indexer))
+      expect(sample["holdings_1display"]).to be_nil
+      expect(sample["location"]).to eq(["Mendel Music Library"])
+    end
+  end
 end
```

Comment:
We want to have a holdings_1display :

"holdings_1display": "
{\"22561746630006421\":
{\"location_code\":\"mendel$qk\",\"location\":\"Remote Storage (ReCAP): Mendel Music Library Use Only\",\"library\":\"Mendel\",\"call_number\":\"LS- 16526\",\"call_number_browse\":\"LS- 16526\",
\"items\":[{\"holding_id\":\"22561746630006421\",\"id\":\"23561746620006421\",\"status_at_load\":\"0\",\"barcode\":\"32101054047947\",\"copy_number\":\"1\"}]}}
"

Reply:
@mzelesky can you also review the holdings_1display that we expect to have? 
the mms_id I used is the one from the fixture: 998370993506421

Reply:
@mzelesky this is the fixture in the PR https://raw.githubusercontent.com/pulibrary/bibdata/a5e9b6f6d0f39de867efea3dfb9f1dfc81b71a62/spec/fixtures/marc_to_solr/998370993506421.mrx

Reply:
@carolyncole I think if you add in the code https://github.com/pulibrary/bibdata/blob/4f7d61c27f547fec6f80549e3717d37e9acb6f6e/marc_to_solr/lib/process_holdings_helpers.rb#L53. You might have added a change already.

``#Select 876 fields (items) with permanent location. 876 location is equal to the 852 permanent location.`
  def select_permanent_location_876(group_876_fields, field_852)
    return group_876_fields if /^scsb/.match?(field_852['b'])
    group_876_fields.select do |field_876| 
      current_location_code(field_876) == permanent_location_code(field_852) || current_location_code(field_876) == 'RES_SHARE$IN_RS_REQ'
    end
  end``

the spec will return holdings_1display:

{\"22561746630006421\":{\"location_code\":\"mendel$qk\",\"location\":\"Remote Storage (ReCAP): Mendel Music Library Use Only\",\"library\":\"Mendel Music Library\",\"call_number\":\"LS- 16526\",\"call_number_browse\":\"LS- 16526\",\"items\":[{\"holding_id\":\"22561746630006421\",\"id\":\"23561746620006421\",\"status_at_load\":\"0\",\"barcode\":\"32101054047947\",\"copy_number\":\"1\"}]}}

Reply:
That holdings_1display looks right.

Reply:
Thanks @mzelesky